packages/commercetools/composables/src/useMakeOrder/index.ts

```diff
@@ -15,7 +15,14 @@ const useMakeOrderFactoryParams = {
 
   make: async (context: Context, { customQuery }): Promise<Order> => {
     const { id, version } = context.cart.cart.value;
-    const response = await context.$ct.api.createMyOrderFromCart({ id, version }, customQuery);
+    // TODO fix type in @vue-storefront/core after update apollo-client
+    const response: any = await context.$ct.api.createMyOrderFromCart({ id, version }, customQuery);
+    if (response.graphQLErrors) {
+      throw response.graphQLErrors[0];
```

Comment:
I believe you want to see a message so it would be better to throw `response.graphQLErrors[0].message`

Reply:
I'm not sure about it. we then log the error and assign it in composable to `error.value.make` to have access to it in components, we want to have access not only to the message or error, but to the error code for example.

also there is a similar example in `packages/commercetools/composables/src/useForgotPassword/index.ts`

Reply:
Ah ok, i get it now